tests/e2e/tests/devfiles/JavaMaven.spec.ts

```diff
@@ -42,7 +42,7 @@ suite(`${stack} test`, async () => {
         commonLsTests.suggestionInvoking(tabTitle, 10, 20, 'append(char c) : PrintStream');
         commonLsTests.errorHighlighting(tabTitle, 'error_text', 11);
         commonLsTests.autocomplete(tabTitle, 10, 11, 'System - java.lang');
-        commonLsTests.codeNavigation(tabTitle, 9, 10, codeNavigationClassName);
+        commonLsTests.codeNavigation(tabTitle, 9, 10, codeNavigationClassName, 30_000); // extended timout to give LS enough time to start
```

Comment:
Wouldn't it be better to extend the timeout for the first test? I see moving the `code navigation` test to line 42 as the easiest solution for that. Applies for all other test files.

Reply:
That would have the opposite effect and the timeout would have to be even longer.
By the time the tests get to codeNavigation the LS is still not initialized.
Moving it to the front would create even bigger unnecessary wait and prolong the test run.

Reply:
So that means that even the tests of LS like `error highlighting`, `autocomplete` etc are failing now, right?

Reply:
for some reason, it's just the code navigation that's being affected.

Reply:
To me, it seems like quite a flaky approach. Also, what about increasing `TimeoutConstants.TS_EDITOR_TAB_INTERACTION_TIMEOUT` instead of changing the code in every test class? The reverting of the workaround should be easier then.

Reply:
I believe it's easier to just remove the custom timeout variable and not change it for all tests including happy path. This way once the tasks exit code is reimplemented, we can just remove the timeout

Reply:
Ok, so leave the comments there so it is clear then :) Thank you for the discussion.